Incorrect stub packaging in latest responses release.

Stubs introduced in recent `responses` release are causing Spurious mypy failures.

### Steps to Reproduce

Given the following file:

```py
# ~/sandbox/responses_bug.py
import responses

responses.add(
    responses.Response(
        method='GET',
        url='http://example.com',
    )
)
```

1. Set up a clean virtual environment (`conda create -n responses-bug python=3.8 pip; conda activate responses-bug; pip install mypy responses`
2. `mypy --ignore-missing-imports --strict ~/sandbox/responses_bug.py`

### Expected Result

I expected mypy to exit successfully with no errors.

### Actual Result

```
.../responses_bug.py:3: error: Call to untyped function (unknown) in typed context
.../responses_bug.py:4: error: Call to untyped function "Response" in typed context
```

According to mypy's `reveal_type`, the revealed_type of `add` is:

```
.../responses_bug.py:3: note: Revealed type is 'def (method: Any =, url: Any =, body: Any =, adding_headers: Any =, *args: Any, **kwargs: Any) -> Any'
```

From what I can tell, mypy is looking at the code, not at the stubs.  Looking in my site-packages confirms that the `__init__.pyi` was not included in the distribution (but `py.typed` _was_ -- which means that `mypy` is looking at the responses _code_, not the responses stubs).  I believe that the fix here is to just update `setup.py` to also include `__init__.pyi` as `package_data` along-side `py.typed`.

--- a/setup.py
+++ b/setup.py
@@ -71,7 +71,7 @@ setup(
     tests_require=tests_require,
     setup_requires=setup_requires,
     cmdclass={"test": PyTest},
-    package_data={"responses": ["py.typed"]},
+    package_data={"responses": ["py.typed", "__init__.pyi"]},
     include_package_data=True,
     classifiers=[
         "Intended Audience :: Developers",

